_newsletters/2020-05-01-may-2020.md

```diff
@@ -0,0 +1,29 @@
+---
+layout: post
+title: May 2020 Newsletter
+category: newsletters
+last_modified_at: 
+tags: newsletters
+newsletter-year: 2020
+sidenav: about_us
+---
+## Your Feedback Requested!
+### Arriving In Your Inbox Tomorrow
+
+Search.gov is always seeking feedback to help us serve our customers better - help us help you!
+
+Tomorrow June 2, please **look out for a survey in your inbox with the subject like “GSA Wants Your Feedback” **sent from <a href="mailto:surveys@research.gsa.gov">surveys@research.gsa.gov</a> - it will have a link to our annual survey. We use these results to better understand how the program is doing, and how we can better meet your needs.
+
+![Screen Shot 2020-06-01 at 2.27.35 PM](https://cdn2.hubspot.net/hub/1962994/hubfs/Screen%20Shot%202020-06-01%20at%202.27.35%20PM.png?upscale=true&width=800&upscale=true&name=Screen%20Shot%202020-06-01%20at%202.27.35%20PM.png) Some users who work with multiple GSA programs may not receive the Search.gov survey. Even if you do not receive the survey, we would still love to hear from you. Please reach out to us by email, and we can send you the questions separately. Thank you in advance! ## System Update ### Upgrades Completed Ahead of Finalizing ATO Renewal We’ve been focused on upgrading our software for [Search.gov](http://search.gov/)'s back end, in anticipation of completing our Authority to Operate (ATO) renewal process. Our most recent upgrades include Elasticsearch, Ubuntu, Rails, and other smaller components to ensure reliability of continued service. 🎉
```

Comment:
do we have a native version of this image?

Reply:
I dont' think so, this is the survey email so wouldn't make sense to put on our site.

Reply:
The image will break in a few months, though - maybe just remove it entirely?

Reply:
removing